### ISSUE TITLE
Telegram crashes when external monitor is disconnected 

### ISSUE THREAD

**Issue by resetnow** [telegramdesktop/tdesktop#2513]
### Steps to reproduce
1. Connect an external monitor to the notebook
2. Open telegram
3. Disconnect the monitor

### Expected behaviour
Telegram does not crash

### Actual behaviour
Telegram crashes

### Configuration
**Operating system:**
Fedora 24

**Version of Telegram Desktop:**
0.10.11

### Logs

```
(Telegram:27744): Gdk-ERROR **: The program 'Telegram' received an X Window System error.
This probably reflects a bug in the program.
The error was 'XI_BadDevice (invalid Device parameter)'.
  (Details: serial 1856 error_code 129 request_code 131 minor_code 48)
  (Note to programmers: normally, X errors are reported asynchronously;
   that is, you will receive the error a while after causing it.
   To debug your program, run it with the --sync command line
   option to change this behavior. You can then get a meaningful
   backtrace from your debugger if you break on the gdk_x_error() function.)
Trace/breakpoint trap (core dumped)
```

**Comment by resetnow** [comment]
abrt backtrace

[backtrace.txt](https://github.com/telegramdesktop/tdesktop/files/517761/backtrace.txt)

**Comment by Aokromes** [comment]
#1179 ?

**Comment by auchri** [comment]
I think it's fix by the move to qt 5.6.2

**Comment by resetnow** [comment]
I'm not sure whether or not it's the same as #1179 — it's not a segfault, and I don't see anything like XI_BadDevice error in that issue.

**Comment by Aokromes** [comment]
Try 0.10.15 alpha it looks they fixed a QT crash.

**Comment by resetnow** [comment]
Yep, fixed in 0.10.16.
